src/noisepy/seis/S1_fft_cc_MPI.py

```diff
@@ -94,18 +96,18 @@ def cross_correlate(
 
     # MPI loop: loop through each user-defined time chunk
     for ick in range(rank, splits, size):
-        t10 = time.time()
         ts = timespans[ick]
         if cc_store.is_done(ts):
             continue
 
         # ###########LOADING NOISE DATA AND DO FFT##################
-        t0 = time.time()
+        t_chunk = tlog.reset()
```

Comment:
More descriptive variable?

Reply:
This is being used to measure the time to process a whole `chunk` (see line 240):
```
tlog.log(f"Process the chunk of {ts}", t_chunk)
```
Seemed descriptive to me? :). What did you have in mind?

Reply:
Gotcha! That makes sense. I would just add a comment with that the first time it's used then.

Reply:
Added comment